spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurations.java

```diff
@@ -157,10 +155,42 @@ static class OpenTelemetryConfiguration {
 
 		@Bean
 		@ConditionalOnMissingBean
-		@ConditionalOnBean(Sender.class)
 		@ConditionalOnEnabledTracing
-		ZipkinSpanExporter zipkinSpanExporter(BytesEncoder<Span> encoder, Sender sender) {
-			return ZipkinSpanExporter.builder().setEncoder(encoder).setSender(sender).build();
+		BytesEncoder<Span> spanBytesEncoder() {
+			return SpanBytesEncoder.JSON_V2;
```

Comment:
allows use of stackdriver, like this https://github.com/openzipkin/zipkin-gcp/blob/10827206f1e4f6b65e261d936ccd1c45e8a42ef9/encoder-stackdriver-brave/src/main/java/zipkin2/reporter/stackdriver/brave/StackdriverV2Encoder.java#L25